packages/@okta/vuepress-theme-prose/enhanceApp.js

```diff
@@ -11,5 +11,15 @@ export default ({
 }) => {
   Vue.component('v-select', VueSelect);
 
+  router.beforeEach((to, from, next) => {
+    /**
+     * Catch `404` in router and redirect to custom 404 page
+     */
+    if (to.matched.length > 0 && to.matched[0].path === "*") {
+      next("/errors/404.html");
+    } else {
+      next();
+    }
+  });
```

Comment:
Just curious, do we not have a central router config file where we can add this mapping?

Reply:
We don't. The behavior here is _kind of_ a hack because Vuepress routing tries to take over from the CloudFront error handling and tries to serve the default `404` page instead. Here we're just forcing it to serve our page instead of the default `404`.

Reply:
We have a `.../404/index.md` file. Does it behave any differently if we remove it ?

Reply:
`errors/404.md` => `errors/404.html` after the site is compiled